### PR TITLE
Improve `engine.create`

### PR DESCRIPTION
**Pull request by bryanforbes**
### Description
Improve `engine.create`. Depends on #90

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [x] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: sqlalchemy-stubs/engine/create.pyi [hunk 1]
```diff
@@ -1,16 +1,146 @@
...
+    case_sensitive: bool = ...,
+    connect_args: Dict[Any, Any] = ...,
+    convert_unicode: bool = ...,
+    creator: Callable[..., Any] = ...,
+    echo: Union[bool, _Debug] = ...,
+    echo_pool: Union[bool, _Debug] = ...,
+    enable_from_linting: bool = ...,
+    encoding: str = ...,
+    execution_options: Dict[Any, Any] = ...,
+    future: Union[bool, Literal[True]],
```
Comment (CaselIT) [inline sqlalchemy-stubs/engine/create.pyi:79]: isn't this just equal to bool? What difference makes `Literal[True]`?

**Review by CaselIT — APPROVED** [review]

File: sqlalchemy-stubs/engine/create.pyi (hunk 1 above)
Comment (bryanforbes) [reply sqlalchemy-stubs/engine/create.pyi]: It is. And evaluating this made me go and check this and I moved the declarations around so it actually works now.

Comment (CaselIT) [reply sqlalchemy-stubs/engine/create.pyi]: oh, so for the overload the precedence matters, like typescript (iirc on ts)

Comment (bryanforbes) [reply sqlalchemy-stubs/engine/create.pyi]: Yes

**Pull request merged**